torch/utils/cpp_extension.py

```diff
@@ -828,6 +828,35 @@ def CUDAExtension(name, sources, *args, **kwargs):
                 cmdclass={
                     'build_ext': BuildExtension
                 })
+
+    Compute capabilities:
+
+    By default the extension will be compiled to run on all archs of the cards visible during the
+    building process of the extension, plus PTX. If down the road a new card is installed the
+    extension may need to be recompiled.
+
+    If a visible card has a compute capability (CC) that's newer than the newest version for which
```

Comment:
I think you should join this sentence with the preceding paragraph so it's clear it's part of the default behavior.